scripts/migration/migration-GPII-3711.js

```diff
@@ -42,7 +42,9 @@ var newValuesForNovaClientCredential = {
         "http://registry.gpii.net/common/language",
         "http://registry.gpii.net/common/DPIScale",
         "http://registry.gpii.net/common/highContrast/enabled",
-        "http://registry.gpii.net/common/selfVoicing/enabled"
+        "http://registry.gpii.net/common/highContrastTheme",
+        "http://registry.gpii.net/common/selfVoicing/enabled",
+        "http://registry.gpii.net/applications/com.microsoft.office"
```

Comment:
I only have this question. "com.microsoft.office" isn't a setting per se like "language" or "DPIScale", it's a solution. Is that okay?

Reply:
As long as this same preference path is used in the save request from Morphic for saving preferences, such as [the otis example](https://github.com/GPII/universal/blob/master/testData/preferences/otis.json5#L9), this should be fine.

Reply:
ok, so in here we handle "paths", no matter whether it's a pref or a solution. Thanks for confirming :)